recipe/thirdpartypasswordless/tplmodels/models.go

```diff
@@ -28,6 +28,7 @@ type User struct {
 	TimeJoined  uint64  `json:"timeJoined"`
```

Comment:
- Also need to change user pagination API to have optional tenant ID in the return type?
- Please also check dashboard related APIs that return the user object to see if tenantId is required in them too.

Reply:
this won't be required now as we decided not to include tenantId in the response sent to frontend.

Reply:
Update the user pagination functions to have tenantId optional. And fill that in in case the user is a third party user with a tenant Id, else it should be nil

Reply:
Done